spec/unit/tracker_spec.lua

```diff
@@ -15,14 +15,16 @@
 -- Copyright:   Copyright (c) 2013 Snowplow Analytics Ltd
 -- License:     Apache License Version 2.0
 
-local tracker = require("tracker")
+local tracker
 
 local collector_uri = "http://d3rkrsqld9gmqf.cloudfront.net/i"
 local TRACKER_VERSION = require("constants").TRACKER_VERSION
 
 describe("tracker", function()
   setup(function()
     _G._TEST = true
+    package.loaded["lua_uuid"] = require("spec/mocks/mock_uuid")
```

Comment:
This is really cool! Is there a reason why it's in the setup callback and not at the beginning of the file as before? I would expect packages to be loaded at the start of the file but either way is fine if there is a reason for it.

Reply:
No technical reason really, it will work at both the top of the file and inside `setup`. I was thinking of it as trying to make it clearer that it's something that is relating to the running of the `describe("tracker"..` test, rather than general code to do with imports.

Happy to move it back if you'd prefer though!